core/src/main/kotlin/Kotlin/DescriptorDocumentationParser.kt

```diff
@@ -107,14 +107,14 @@ class DescriptorDocumentationParser @Inject constructor(
      * *@attr ref <android.>R.styleable.<attribute_name>
      * *@attr name <attribute_name>
      * *@attr description <attribute_description>
-     * This also adds the @since tag.
+     * This also adds the @since and @apiSince tags.
      */
     private fun DocumentationNode.addExtraTags(tags: Array<KDocTag>, descriptor: DeclarationDescriptor) {
         tags.forEach {
             val name = it.name
             if (name?.toLowerCase() == "attr") {
                 it.getAttr(descriptor)?.let { append(it, RefKind.Detail) }
-            } else if (name?.toLowerCase() == "since") {
+            } else if (name?.toLowerCase() == "since" || name?.toLowerCase() == "apisince") {
```

Comment:
Oh that's interesting. Normally I'd imagine that we'd modify the previous commit so that in case anyone wants to browse the repository in the future they'll have fewer commits to look through to find what they're looking for. I think you'd have to `git push -f` in order for git to permit doing that though.

Although I guess if there were lots of code I'm not sure if that would make it hard to see what changed between patchset versions - the last time I checked, GitHub would just delete old patchsets rather than display them (I think GitHub doesn't model the concept of a patchset).

The code looks reasonable to me so I'm supportive of submitting it.

Reply:
Thanks for the review!

Yep - I usually submit separate commits during review and then squash + merge when merging to the main branch.

Reply:
That's a good idea